Multiple reportportal plugin improvements and fixes (#3356)
teemtee/tmt: docs/releases.rst

``rlPhaseEnd`` saved in ``results.yaml`` are now relative to the
``execute`` directory.

The :ref:`/plugins/report/reportportal` plugin now handles the
timestamps for ``custom`` and ``restraint`` results correctly. It

> should never happen that the result's ``start-time`` will be

Reviewer: ```suggestion
should prevent the ``start-time`` of a result being higher 
```

Author: Fixed in https://github.com/teemtee/tmt/pull/3356/commits/5aa14401a4ca5aa143e4b976b7719b73ef3d6e96

Thanks
